.github/workflows/deploy.yml

```diff
@@ -75,7 +75,10 @@ jobs:
       id: changed-egress-config
       uses: tj-actions/changed-files@v34
       with:
-        files: deploy-config/egress_proxy/notify-api-staging.*.acl
+        files: |
+          deploy-config/egress_proxy/notify-api-staging.*.acl
+          .github/actions/deploy-proxy/action.yml
+          .github/workflows/deploy.yml
```

Comment:
This last line will let us trigger a deploy by updating deploy-proxy call in this file by adding

```
with:
  proxy_version: GIT_SHA
```